pkg/cmd/triggertemplate/list.go

```
@@ -0,0 +1,132 @@
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package triggertemplate
+
+import (
+	"fmt"
+	"os"
+	"text/tabwriter"
+
+	"github.com/spf13/cobra"
+	"github.com/tektoncd/cli/pkg/cli"
+	"github.com/tektoncd/cli/pkg/formatted"
+	"github.com/tektoncd/cli/pkg/helper/validate"
+	"github.com/tektoncd/cli/pkg/printer"
+	"github.com/tektoncd/triggers/pkg/apis/triggers/v1alpha1"
+	"github.com/tektoncd/triggers/pkg/client/clientset/versioned"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	cliopts "k8s.io/cli-runtime/pkg/genericclioptions"
+)
+
+const (
+	emptyMsg = "No triggertemplates found"
+)
+
+func listCommand(p cli.Params) *cobra.Command {
+	f := cliopts.NewPrintFlags("list")
+
+	eg := `
+# List all triggertemplates in namespace 'bar'
+tkn trigger list -n bar
+`
+
+	c := &cobra.Command{
+		Use:     "list",
+		Aliases: []string{"ls"},
+		Short:   "Lists triggertemplates in a namespace",
+		Annotations: map[string]string{
+			"commandType": "main",
+		},
+		Example: eg,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if err := validate.NamespaceExists(p); err != nil {
+				return err
+			}
+
+			cs, err := p.Clients()
+			if err != nil {
+				return err
+			}
+
+			tts, err := list(cs.Triggers, p.Namespace())
+			if err != nil {
+				fmt.Fprintf(os.Stderr, "Failed to list triggertemplates from %s namespace \n", p.Namespace())
+				return err
+			}
+
+			output, err := cmd.LocalFlags().GetString("output")
+			if err != nil {
+				fmt.Fprint(os.Stderr, "Error: output option not set properly \n")
+				return err
+			}
+
+			stream := &cli.Stream{
+				Out: cmd.OutOrStdout(),
+				Err: cmd.OutOrStderr(),
+			}
+
+			if output != "" {
+				return printer.PrintObject(stream.Out, tts, f)
+			}
+
+			if err = printFormatted(stream, tts, p); err != nil {
+				fmt.Fprint(os.Stderr, "Failed to print triggertemplates \n")
+				return err
```

Maybe just return the error here instead of using `os.Stderr`? 

```go
return errors.New("failed to print triggertemplates\n"
```